core/classes/Discord.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/*
+ *	Made by Aberdeener
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.0-pr8
+ *
+ *  License: MIT
+ *
+ *  Discord class
+ */
+class Discord {
+
+    public static function discordBotRequest($url = '/') {
+
+        $bot_url_attempt = Util::curlGetContents(BOT_URL . $url);
+
+        switch ($bot_url_attempt) {
```

Comment:
Could this be simplified using if (in_array(...)), instead of comparing cases and just returning them?
eg
```php
if (in_array($bot_url_attempt, array('success', 'failure-cannot-interact', 'failure-invalid-api-url'))) {
  return $bot_url_attempt;
}

// Backup attempt goes here
```

Reply:
Good point, fixed